deploy.sh

```diff
@@ -21,6 +21,9 @@ do
   sleep 1
 done
 
+# Give route a chance to be ready
+sleep 5
```

Comment:
Should we add a check which keeps looking at the route status and sleeps till it's ready?